api/cluster/templater.go

```diff
@@ -205,6 +216,43 @@ func createLoggerSpec(loggerURL string, loggerConfig models.LoggerConfig) *kfsv1
 	}
 }
 
+func createCustomPredictorSpec(modelService *models.Service, resources v1.ResourceRequirements) kfsv1alpha2.PredictorSpec {
+	envVars := modelService.EnvVars
+	envVars = append(envVars, models.EnvVar{Name: envPredictorPort, Value: defaultPredictorPort})
+	envVars = append(envVars, models.EnvVar{Name: envPredictorModelName, Value: modelService.Name})
+	envVars = append(envVars, models.EnvVar{Name: envPredictorArtifactLocation, Value: defaultPredictorArtifactLocation})
+	envVars = append(envVars, models.EnvVar{Name: envPredictorStorageURI, Value: utils.CreateModelLocation(modelService.ArtifactURI)})
```

Comment:
As discussed offline, we can omit `STORAGE_URI` if user doesn't specify the artifact URI.

Reply:
it is already updated here http://custom-wo-artifact-17.integration-test.models.id.d.gods.golabs.io/v1/models/custom-wo-artifact-17:predict